indexing-solr/src/main/java/edu/unc/lib/boxc/indexing/solr/filter/SetDescriptiveMetadataFilter.java

```diff
@@ -282,6 +283,31 @@ private void extractSubjects(Element mods, IndexDocumentBean idb) {
 
     }
 
+    private void extractOtherSubjects(Element mods, IndexDocumentBean idb) {
+        List<Element> otherSubjectEls = mods.getChildren("subject", JDOMNamespaceUtil.MODS_V3_NS);
+        List<String> otherSubjects = new ArrayList<>();
+        if (!otherSubjectEls.isEmpty()) {
+            for (Element otherSubjectObj : otherSubjectEls) {
+                List<Element> subjectParts = otherSubjectObj.getChildren();
+                for (Element otherSubject : subjectParts) {
+                    String otherSubjectName = otherSubject.getName();
+                    if (otherSubjectName.equals("name") || otherSubjectName.equals("topic")) {
+                        continue;
+                    }
+
+                    if (otherSubject.getChildren().isEmpty()) {
```

Comment:
Just a thought, but it might make sense to merge this conditional with the one at 194 since they are all conditions if which we want to skip adding the contents of this subject